### PR TITLE
update pep8 to 1.6.1

### PR DESCRIPTION
**Pull request by magopian**
Fix new warnings raised by pep8: https://pypi.python.org/pypi/pep8/1.6.1#id2

### CODE REVIEW

File: scripts/sign_all.py [hunk 1]
```diff
@@ -1,43 +0,0 @@
-#!/usr/bin/env python
```
Comment (diox) [inline scripts/sign_all.py]: mmm ? why is this file removed ?

Comment (magopian) [inline scripts/sign_all.py]: I was going to fix the module-level imports not at the top of the file, and then realized that it's been replaced by https://github.com/mozilla/olympia/blob/master/apps/addons/management/commands/process_addons.py#L30 (to sign all addons) and https://github.com/mozilla/olympia/blob/master/apps/addons/management/commands/sign_addons.py for specific addons.

File: apps/addons/decorators.py [hunk 2]
```diff
@@ -14,9 +14,12 @@ def addon_view(f, qs=Addon.objects.all):
     def wrapper(request, addon_id=None, app_slug=None, *args, **kw):
         """Provides an addon given either an addon id or an addon slug."""
         assert addon_id, 'Must provide addon id or slug'
-        get = lambda **kw: get_object_or_404(qs(), **kw)
+
+        def get_or_404(**kw):
```
Comment (diox) [inline apps/addons/decorators.py]: Is this function actually useful ? I'd just call `get_object_or_404(qs(), ...)` in both cases and get rid of it.

File: apps/api/__init__.py [hunk 3]
```diff
@@ -1,10 +1,11 @@
+from django.db.models.signals import post_save, post_delete
+from piston.models import consumer_post_save, consumer_post_delete, Consumer
```
Comment (diox) [inline apps/api/__init__.py]: Add newline since we put django in its own import group

File: scripts/sign_all.py (hunk 1 above)
Comment (diox) [inline scripts/sign_all.py]: I'd remove it in a separate commit/pull request (that we can merge before this one)

**Comment by diox** [comment]
r+wc

File: scripts/sign_all.py (hunk 1 above)
Comment (magopian) [inline scripts/sign_all.py]: Did that in PR #452
